apps/examples/wc-next10/pages/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Head from 'next/head';
+import dynamic from 'next/dynamic'
+
+const PButton = dynamic(() => import('../ButtonLogic'), { ssr: false });
```

Comment:
is there a reason why we are setting ssr to false here?

Reply:
yep, if I set this to true I get a `window is not defined` error I think because it tries to render the component before a window exists. I've looked into this and it was between this option or using the UseEffect hook? In the ticket it said we'd look into ssr later down the line so I thought I'd go with this option to get it rendering for now. Happy to jump on a call if you have any other suggestions though 😄

Reply:
I see you already mentioned the reason on top, but that means the button will not render on the server. is this expected for now?

Reply:
I see, I think your solution is the cleanest. but my main concern is that we aren't SSR-ing the component which might be a bottleneck for many of our apps. happy to continue the conversation down the line when we need to look into this but from quick research, it seems the lit team has just released a wrapper for nextjs that we might want to consider. 

https://github.com/lit/lit/tree/main/packages/labs/nextjs

Reply:
@raoufswe I believe handling SSR is a separate piece of work coming up very soon :)